pkg/avatar/initials_svgo.go

```diff
@@ -0,0 +1,49 @@
+package avatar
+
+import (
+	"bytes"
+	"context"
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	svg "github.com/ajstarks/svgo"
+)
+
+// SVGoInitials create SVG avatars with initials in it.
+//
+// This implementation is based on the `svgo` library.
+type SVGoInitials struct {
+}
+
+// NewSVGoInitials instantiate a new [SVGoInitials].
+func NewSVGoInitials() *SVGoInitials {
+	return &SVGoInitials{}
+}
+
+// ContentType return the generated avatar content-type.
+func (a *SVGoInitials) ContentType() string {
+	return "image/svg+xml"
+}
+
+// Generate will create a new avatar with the given initials and color.
+func (a *SVGoInitials) Generate(ctx context.Context, initials, color string) ([]byte, error) {
+	var buf bytes.Buffer
+
+	b, err := hex.DecodeString(strings.TrimPrefix(color, "#"))
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse the color: %w", err)
+	}
+
+	fontSize := 128
+
+	width := 128
+	height := 128
```

Comment:
I think these should be attributes of the struct and passed to `NewSVGoInitials`.

Reply:
At the moment we only need one size and I'm really not sure that the icon will behave correctly if we change those values.